benchmark/test_binary_pointwise_perf.py

```diff
@@ -50,6 +50,7 @@ def get_tflops(self, op, *args, **kwargs):
             ("floor_divide", torch.floor_divide, INT_DTYPES),
             ("remainder", torch.remainder, INT_DTYPES),
             ("rsub", torch.rsub, FLOAT_DTYPES),
+            ("logical_or", torch.logical_or, FLOAT_DTYPES),
```

Comment:
I suggest only test logical op on int and bool dtypes

Reply:
Fixed thx